corehq/apps/app_manager/static/app_manager/js/app_manager.js

```diff
@@ -296,6 +300,40 @@ hqDefine('app_manager/js/app_manager.js', function () {
                 $(related).data(name, value);
             });
         }
+        function resetIndexes($sortable) {
```

Comment:
it's still necessary because the index values have at least been updated. updateprop doesn't have to be in there tho